app/views/widgets/shared/_omninav.html

```diff
@@ -302,7 +302,7 @@
                           <a href="https://blogs.chapman.edu/business/" title="Argyros School of Business &amp; Economics">Argyros School of Business &amp; Economics</a>
                         </li>
                         <li>
-                          <a href="https://blogs.chapman.edu/ces/" title="College of Educational Studies">College of Educational Studies</a>
+                          <a href="https://blogs.chapman.edu/ces/" title="Attallah College of Educational Studies">College of Educational Studies</a>
```

Comment:
No, thanks you're right, I missed that one. We're leaving the url of /ces because that's not changed for blogs yet but I only changed the title in the above link and forgot the visible label. Thanks I will fix